en/companion_computer/holybro_pixhawk_jetson_baseboard.md

```diff
@@ -0,0 +1,1074 @@
+
+![SDK Manager installation components page](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/nvidia_sdkmanager_2.png)
+
+Next confirm your selected device:
+
+- Choose `Pre-config` for the OEM Configuration (this will skip Ubuntu first time setup screens after reboot).
+- Choose your preferred username and password (and write them down).
+  These will be used as your login credentials to Jetpack.
+- Choose `NVMe` as the storage device because the board has separate SSD for storage.
+
+![SDK Manager installation storage and OEM config page](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/nvidia_sdkmanager_3.png)
+
+Press **Flash** to start installing the image.
+It will take several minutes for the installation to complete.
+
+::: warning
+The fan will be running while the installation is going on, so you may make sure it not blocked.
+Jetson will boot into initial login after flashing.
+:::
+
+## Jetson Network Setup
+
+After flashing the Jetson it will reboot to the login screen (skipping recovery mode).
+
+The diagram below shows how you can connect your Jetson carrier board for the first time to setup the network connection.
+This step is needed so that in future we can connect to the Jetson via SSH instead of using an external monitor (if we want).
+
+Connect the external monitor to the board's (Jetson) HDMI port and the Jetson to power via the XT-30 input.
+Separately connect the Pixhawk power supply using either the USB-C (on the side of the FC) or the `Power1`/`Power2` ports next to the Jetson.
+
+::: tip
+Pixhawk also has to be powered because the internal Ethernet switch on the board is not powered by XT30 connector.
+:::
+
+![First time network setup connection diagram and power](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/network_diagram.png)
+
+Connect the keyboard and mouse via spare USB ports.
+The external monitor should display the login screen.
+Enter the username and password you set when using the SDKManager to flash the Jetson.
+
+Next open a terminal in order to get the IP address of the Jetson.
+This can be used later to connect via SSH.
+Enter the following command.
+
+```sh
+ip addr show
+```
+
+The output should look similar to this, indicating (in this case) an IP address of `192.168.1.190`
+
+```sh
+
+```
+
+::: tip If the command doesn't work ...
+The WiFi card with the board is either the Intel 8265NGW AC Dual Band or Realtek RTL8B22CE.
+The Intel module may not work out of the box, in which case you should open a terminal and run the following command to install its driver.
+
+```sh
+sudo apt-get install -y backport-iwlwifi-dkms
+```
+
+Then repeat the process try to get the IP address again.
+
+:::
+
+Now that we have an IP address, open a terminal in your _development computer_.
+We can log into the Jetson using the IP address, as shown:
+
+```sh
+ssh holybro@192.168.1.190 #(Your defined IP might be different)
```

Comment:
@farhangnaderi When I do this, do I need to enter my credentials? Same ones as before?